core/services/workflows/engine.go

```diff
@@ -715,9 +720,15 @@ func (e *Engine) worker(ctx context.Context) {
 				continue
 			}
 
+			cma := e.cma.With(eIDKey, executionID)
+			logCustMsg(cma, "starting execution on a trigger event", e.logger)
```

Comment:
@vyzaldysanchez I think we can remove this and just have the two below for starting + failed to start an execution.